### ISSUE TITLE
Support CPE scanning. 

### ISSUE THREAD

**Issue by oliverchang** [google/osv-scanner#353]

**Comment by andrewpollock** [comment]
@oliverchang can you update the description of this with what your thoughts were here?

**Comment by oliverchang** [comment]
> @oliverchang can you update the description of this with what your thoughts were here?

done!

**Comment by andrewpollock** [comment]
Is your thinking to translate CPEs to Purls or ESIs here, in terms of what the OSV.dev API winds up being called with?

**Comment by oliverchang** [comment]
Yes, that was my original thinking, but the exact implementation details are TBD if we do decide to do this. This translation could also be done at the OSV.dev API level.
